pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -83,10 +85,14 @@ func newClusterResourceOverride(client clientset.Interface, config io.Reader) (a
 	}, nil
 }
 
-func wrapLimit(limitRange *kapi.LimitRange, resourceName string, obj runtime.Object) error {
-	// limitranger.Limit(limitRange, resourceName, obj)
```

Comment:
Yeah sorry that's replaced by the new code - it's just stuff to feed to LimitRanger to not have it do any limiting. Didn't compile? That's weird...